sign/sign.go

```diff
@@ -66,12 +70,56 @@ func (s *Signer) UploadBlob(path string) error {
 func (s *Signer) SignImage(reference string) (*SignedObject, error) {
 	s.log.Infof("Signing reference: %s", reference)
 
-	// TODO: unimplemented
+	if s.options.Experimental {
+		os.Setenv("COSIGN_EXPERIMENTAL", "true")
+	}
+
+	ko := sign.KeyOpts{
+		KeyRef:     s.options.KeyPath,
+		PassFunc:   generate.GetPass,
+		FulcioURL:  s.options.FulcioURL,
+		RekorURL:   s.options.RekorURL,
+		OIDCIssuer: s.options.OIDCIssuer,
+
+		InsecureSkipFulcioVerify: false,
+	}
+	regOpts := cliOpts.RegistryOptions{
+		AllowInsecure: true,
+	}
```

Comment:
I would say we should not allow this for now.

Reply:
should we make this a config option?

Reply:
made as config option